dotCMS/src/main/webapp/html/portlet/ext/browser/view_browser_js_inc.jsp

```diff
@@ -944,7 +947,10 @@ Structure defaultFileAssetStructure = CacheLocator.getContentTypeCache().getStru
                 Event.observe(asset.inode + '-TR', 'mouseover', mouseOverContent);
 
                 if((!asset.live && write) || (asset.live && publish)) {
-                    var draggable = new Draggable(asset.inode + '-DIV', { ghosting:true, revert:true, zindex: 1000 });
+                    // reverteffect -> It's need it to return the element to its place
+                    // We can't use the reverteffect function librery has by default because the initial element doesn't have
+                    // an initial value of top_offset or left_offset.

```

Comment:
```suggestion
                    /*
                        reverteffect: Needs to return the element to its place.
                        We can't use the default reverteffect function because the initial element doesn't have an initial 
                        value of top_offset or left_offset.
                    */
```